talk/basicconcepts/coresyntax.tex

```diff
@@ -144,20 +143,19 @@
 
 \begin{frame}[fragile]
   \frametitlecpp[98]{Useful aliases}
-    \alert{Requires inclusion of headers}
   \begin{cppcode}
-    #include <cstddef> // and many other headers
+    #include <cstddef> // (and others) defines these:
 
-    size_t s = sizeof(int); // unsigned integer
+    std::size_t s = sizeof(int); // unsigned integer
                             // can hold any variable's size
 
-    #include <cstdint>
+    #include <cstdint> // defines these:
 
-    ptrdiff_t c = &s - &s;  // signed integer, can hold any
+    std::ptrdiff_t c = &s - &s;  // signed integer, can hold any
                             // diff between two pointers
```

Comment:
Does not fit anymore in the line.
The comment should be put before the code

Reply:
done